nsv13/code/modules/clothing/custom_clothes.dm

```diff
@@ -275,30 +275,26 @@
 	worn_icon = 'nsv13/icons/mob/head.dmi'
 	icon_state = "hardsuit1-peacekeeper_space"
 	item_state = "peacekeeper_space"
-	worn_icon_state = "peacekeeper_space"
+	hardsuit_type = "peacekeeper_space"
 	desc = "A hardsuit helmet fitted with highly experimental magnetic interlocks, allowing it to create a vacuum seal around the user, permitting usage in a hard vacuum. It is currently in EVA mode."
 	alt_desc = "A hardsuit helmet fitted with highly experimental magnetic interlocks, allowing it to create a vacuum seal around the user, permitting usage in a hard vacuum. It is currently in IVA mode."
-	armor = list("melee" = 20, "bullet" = 30, "laser" = 10, "energy" = 10, "bomb" = 70, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 15,"energy" = 30, "bomb" = 25, "bio" = 100, "rad" = 50, "fire" = 75, "acid" = 75, "stamina" = 50)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT //we want to see the mask
-	heat_protection = HEAD
-	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 
 /obj/item/clothing/suit/space/hardsuit/syndi/peacekeeper
 	name = "SG-1 Mjolnir Armour"
 	desc = "An extremely bulky suit of armour fitted with highly experimental magnetic interlocks, allowing it to create a vacuum seal around the user, permitting usage in a hard vacuum. It is currently in EVA mode."
 	alt_desc = "An extremely bulky suit of armour fitted with highly experimental magnetic interlocks, allowing it to create a vacuum seal around the user, permitting usage in a hard vacuum. It is currently in IVA mode."
 	icon = 'nsv13/icons/obj/clothing/suits.dmi'
 	worn_icon = 'nsv13/icons/mob/suit.dmi'
-	icon_state = "peacekeeper_space"
+	icon_state = "hardsuit1-peacekeeper_space"
 	item_state = "peacekeeper_space"
-	worn_icon_state = "peacekeeper_space"
+	hardsuit_type = "peacekeeper_space"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi/peacekeeper
-	armor = list("melee" = 40, "bullet" = 60, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 40, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 15, "energy" = 30, "bomb" = 25, "bio" = 100, "rad" = 50, "fire" = 75, "acid" = 75, "stamina" = 50)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
-	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
-	var/stomp_cooldown_time = 0.3 SECONDS
+	var/stomp_cooldown_time = 0.5 SECONDS
 	var/current_cooldown = 0
 
```

Comment:
Inheritance gore, doublecheck all things that are being inherited from the syndicate hardsuit.

Including, but not limited to:

- Remove ILLEGAL from item_flags
- doublecheck the things in allowed
- syndisuit is weight class normal which fits in backpacks and just about anywhere.
- syndisuit is artifact-proof

I also do not think a ZERO slowdown on-demand hardsuit is good, which the suit toggle gives you in combatmode. Hardsuits that the crew has access to should always have at least some level of drawback. Should be at least 25% or so of base value in my opinion.

Reply:
Ah... I haven't checked over the old gore KMC left... my bad

Reply:
@SerynEngi I have pretty much 0 clue how to do any of this you want me to do except change weight class to bulky.

Reply:
The suit should have some level of slowdown even in speed form, I also believe the syndie and crew suits should be roughly on par as this is a combat vessel.. they are expected to tussle with the syndicate. Regardless, please add some slow down, less friction to make something weak and buff it than it is to make it strong then nerf it. I'm sure Delta or Bokkie can weigh in at what a reasonable slowdown for a 'speedy' mode hardsuit would be if you ask them.